usr/sbin/p11sak/p11sak.c

```diff
@@ -5738,9 +5762,14 @@ static CK_RV handle_obj_list(CK_OBJECT_HANDLE key, CK_OBJECT_CLASS class,
 
 static CK_RV p11sak_list_obj_compare(CK_OBJECT_HANDLE obj1,
                                      CK_OBJECT_HANDLE obj2,
-                                     int *result, void *private)
+                                     int *result,
+                                     void *private)
 {
+#if defined(_AIX)
+    struct p11sak_list_data *data = global_private;
+#else
     struct p11sak_list_data *data = private;
+#endif
```

Comment:
This change is now redundant, because line 4705 takes care of the parameter that gets passed to `iterate_compare_aix`.